DataBase/db.go

```diff
@@ -0,0 +1,77 @@
+package database
+
+import (
+	"main/repo"
+
+	"github.com/jmoiron/sqlx"
+	_ "github.com/lib/pq"
+)
+
+func NewDB() (*RecordDB, error) {
+	connStr := "postgresql://postgres:password@database:5432/postgres?sslmode=disable"
+	db, err := sqlx.Open("postgres", connStr)
+	if err != nil {
+		return nil, err
+	}
+
+	err = db.Ping()
+	if err != nil {
+		return nil, err
+	}
+
+	return &RecordDB{
+		DB: db,
+	}, nil
+}
+
+type RecordDB struct {
+	*sqlx.DB
+}
+
+func NewRecordDB(db *sqlx.DB) *RecordDB {
+	return &RecordDB{
+		DB: db,
+	}
+}
+func (db *RecordDB) NewRecord(r *repo.Record) error {
+	err := db.Get(r, `INSERT INTO records VALUES ($1, $2, $3, $4, $5, $6) RETURNING *`, r.Id, r.Type, r.CaesarShift, r.Result, r.Created_at, r.Updated_at)
```

Comment:
1. move query to constant
2. specify columns you insert to: `INSERT INTO records (col1, col2, ...) VALUES`
3. do not use `RETURNING *` - specify fields you need